azext_iot/_params.py

```diff
@@ -495,6 +496,38 @@ def load_arguments(self, _):
             "Account and Contributor role for the IoT Hub.",
         )
 
+    with self.argument_context("iot hub device-identity generate-mqtt-credentials") as context:
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "--cs"],
+            help="Target connection string. This bypasses the IoT Hub registry and generates the user credentials directly "
+                 "from the supplied symmetric key without further validation. All other command parameters aside from "
+                 "password-creation-time and password-expiry-in-secs will be ignored. Supported connection string types: Device, "
+                 "Module."
+        )
+        context.argument(
+            "duration",
+            options_list=["--duration", "--du"],
+            type=int,
+            help="Valid password duration in seconds.",
```

Comment:
nit, something like "Indicates the duration in seconds, for which the mqtt credentials will remain valid."